crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -58,9 +58,7 @@ def upgrade_error_flash(message)
     end
 
     def upgrade_ha_repo_needed?
-      return true if Proposal.where(barclamp: "pacemaker").first
-
-      false
+      !!Proposal.where(barclamp: "pacemaker").first
```

Comment:
Ok, my aim was to make it simpler, really. I will offer an alternative (I still have some) and update the PR.